src/app/services/copiedForGitHubCommenting.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { map, flatMap, catchError } from 'rxjs/operators';
+import { throwError, of } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WorldChampionsService {
+
+  initialYear = 1950;
+  api = 'http://ergast.com/api/f1';
+
+  constructor(
+    private http: HttpClient
+  ) { }
+
+  getWorldChampions(from: number, to: number) {
```

Comment:
Do you want your methods and properties to be available outside of the class? If yes, id mark them as `public` explicitly, if no, I'd mark them as `private`, and if you want them to be `private` but able to be inherited by deriving classes, I'd mark them as `protected`. 

I'd do this for everything, all methods, and properties (including `initialYear` and `api`)

[The docs probably explain this better](https://www.typescriptlang.org/docs/handbook/classes.html#public-private-and-protected-modifiers)

Reply:
Of course if you dont mark with private/public/protected, everything just defaults to public. I like explicit for maintainability and making your code clear for other coders though. Its a style thing, and no everyone will agree. :)